<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HyperboleFitWidget</class>
 <widget class="QWidget" name="HyperboleFitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1090</width>
        <height>588</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListView" name="qlv_analytes">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="qpb_addAnalyte">
             <property name="text">
              <string>Add analyte</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpb_removeAnalyte">
             <property name="text">
              <string>Remove analyte</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListView" name="qlv_concentrations">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="qpb_addConcentration">
             <property name="text">
              <string>Add concentration</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpb_removeConcentration">
             <property name="text">
              <string>Remove concentration</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListView" name="qlv_mobilities">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="qpb_addMobility">
             <property name="text">
              <string>Add mobility</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpb_removeMobility">
             <property name="text">
              <string>Remove mobility</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="ql_fitMode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Fit mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="qcbox_fitMode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QGroupBox" name="qgb_singleFit">
               <property name="title">
                <string>Single fit (for first peak)</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QFormLayout" name="formLayout">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetMinimumSize</enum>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="ql_analyteAName">
                    <property name="text">
                     <string>Analyte:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="qle_analyteAName">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="1" column="0">
                   <widget class="QLabel" name="ql_csMobility">
                    <property name="text">
                     <string>μ (AS)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="FloatingValueLineEdit" name="qle_csMobility">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="ql_freeMobility">
                    <property name="text">
                     <string>μ (A)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="FloatingValueLineEdit" name="qle_KCS">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="ql_KCS">
                    <property name="text">
                     <string>K (AS)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="FloatingValueLineEdit" name="qle_freeMobility">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QMappedCheckBox" name="qcb_fixedFreeMobility">
                    <property name="text">
                     <string>Fixed</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QMappedCheckBox" name="qcb_fixedCsMobility">
                    <property name="text">
                     <string>Fixed</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QMappedCheckBox" name="qcb_fixedKCS">
                    <property name="text">
                     <string>Fixed</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="qgb_doubleFit">
               <property name="title">
                <string>Double fit (for second peak)</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QFormLayout" name="formLayout_2">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetMinimumSize</enum>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="ql_analyteBName">
                    <property name="text">
                     <string>Analyte:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="qle_analyteBName">
                    <property name="frame">
                     <bool>true</bool>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="2" column="1">
                   <widget class="FloatingValueLineEdit" name="qle_KCSB">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="FloatingValueLineEdit" name="qle_freeMobilityB">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="ql_KCS_2">
                    <property name="text">
                     <string>K (AS)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="ql_freeMobilityB">
                    <property name="text">
                     <string>μ (A)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="ql_csMobilityB">
                    <property name="text">
                     <string>μ (AS)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="FloatingValueLineEdit" name="qle_csMobilityB">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QMappedCheckBox" name="qcb_fixedFreeMobilityB">
                    <property name="text">
                     <string>Same as first analyte</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QMappedCheckBox" name="qcb_fixedCsMobilityB">
                    <property name="text">
                     <string>Same as first analyte</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QMappedCheckBox" name="qcb_fixedKCSB">
                    <property name="text">
                     <string>Same as first analyte</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="qcb_swapAnalytes">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Swap analytes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="qgb_fitParams">
           <property name="title">
            <string>Fitter parameters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="4" column="0">
               <widget class="QLabel" name="ql_iterations">
                <property name="text">
                 <string>Total iterations:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="ql_maxIters">
                <property name="text">
                 <string>Maximum nubmer of iterations</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="FloatingValueLineEdit" name="qle_epsilon">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="qle_maxIters">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="ql_rss">
                <property name="text">
                 <string>S</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="FloatingValueLineEdit" name="qle_rss">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="ql_epsilon">
                <property name="text">
                 <string>ε</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="ql_visositySlope">
                <property name="text">
                 <string>Viscosity slope</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="FloatingValueLineEdit" name="qle_viscositySlope"/>
              </item>
              <item row="4" column="1">
               <widget class="FloatingValueLineEdit" name="qle_iterations">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="qpb_estimate">
                <property name="text">
                 <string>Estimate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="qpb_fit">
                <property name="text">
                 <string>Fit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="qpb_redrawPoints">
                <property name="text">
                 <string>Redraw</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="qgb_statistics">
              <property name="title">
               <string>Statistics</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QComboBox" name="qcbox_statData"/>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="qpb_leftConfIntr">
                   <property name="text">
                    <string>Left</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="qpb_rightConfIntr">
                   <property name="text">
                    <string>Right</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QPushButton" name="qpb_bothConfIntr">
                   <property name="text">
                    <string>Both</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="2" column="0">
                  <widget class="QLabel" name="ql_chartVerticalAMarker">
                   <property name="text">
                    <string>Chart vertical marker A</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <widget class="QLineEdit" name="qle_chartVerticalAMarker">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="qcb_chartVerticalAMarker">
                     <property name="text">
                      <string>Show</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="qpb_chartVerticalAMarkerItsc">
                     <property name="text">
                      <string>Intersection</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="ql_chartVerticalBMarker">
                   <property name="text">
                    <string>Chart vertical marker B</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="ql_statUnits">
                   <property name="text">
                    <string>Units</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="qcbox_statUnits"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="ql_chartHorizontalMarker">
                   <property name="text">
                    <string>Chart horizontal marker</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLineEdit" name="qle_chartHorizontalMarker">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="qcb_chartHorizontalMarker">
                     <property name="text">
                      <string>Show</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="3" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <widget class="QLineEdit" name="qle_chartVerticalBMarker">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="qcb_chartVerticalBMarker">
                     <property name="text">
                      <string>Show</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="qpb_chartVerticalBMarkerItsc">
                     <property name="text">
                      <string>Intersection</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatingValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/gui/common/floatingvaluelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QMappedCheckBox</class>
   <extends>QCheckBox</extends>
   <header>src/gui/common/qmappedcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>qlv_analytes</tabstop>
  <tabstop>qpb_addAnalyte</tabstop>
  <tabstop>qpb_removeAnalyte</tabstop>
  <tabstop>qlv_concentrations</tabstop>
  <tabstop>qpb_addConcentration</tabstop>
  <tabstop>qpb_removeConcentration</tabstop>
  <tabstop>qlv_mobilities</tabstop>
  <tabstop>qpb_addMobility</tabstop>
  <tabstop>qpb_removeMobility</tabstop>
  <tabstop>qcbox_fitMode</tabstop>
  <tabstop>qle_analyteAName</tabstop>
  <tabstop>qle_freeMobility</tabstop>
  <tabstop>qle_csMobility</tabstop>
  <tabstop>qle_KCS</tabstop>
  <tabstop>qcb_fixedFreeMobility</tabstop>
  <tabstop>qcb_fixedCsMobility</tabstop>
  <tabstop>qcb_fixedKCS</tabstop>
  <tabstop>qle_analyteBName</tabstop>
  <tabstop>qle_freeMobilityB</tabstop>
  <tabstop>qle_csMobilityB</tabstop>
  <tabstop>qle_KCSB</tabstop>
  <tabstop>qcb_fixedFreeMobilityB</tabstop>
  <tabstop>qcb_fixedCsMobilityB</tabstop>
  <tabstop>qcb_fixedKCSB</tabstop>
  <tabstop>qle_viscositySlope</tabstop>
  <tabstop>qle_maxIters</tabstop>
  <tabstop>qle_epsilon</tabstop>
  <tabstop>qle_rss</tabstop>
  <tabstop>qle_iterations</tabstop>
  <tabstop>qpb_estimate</tabstop>
  <tabstop>qpb_fit</tabstop>
  <tabstop>qpb_redrawPoints</tabstop>
  <tabstop>qcbox_statData</tabstop>
  <tabstop>qpb_leftConfIntr</tabstop>
  <tabstop>qpb_rightConfIntr</tabstop>
  <tabstop>qpb_bothConfIntr</tabstop>
  <tabstop>qcbox_statUnits</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
